### ISSUE TITLE
Common Functions

### ISSUE THREAD

**Issue by abruns123** [ubsuny/CP1-24-HW2#89]
I think it may be useful at the beginning of a project or hw to discuss more about common functions that we'll all need for our code. For example, my pVectorGenerator file included a function called vectorGen which produced a random vector. If anyone else had needed to produce randomized vectors, they could easily import the file rather than making their own function for it. Doing something like that more in the future may help boost our efficiency as I noticed that many people made their own matrix generators instead of just one function in a separate file that everyone could import. Just want to stir up some discussion about this.

**Comment by ojha-aditya** [comment]
I agree. I think for future homework tasks it would be helpful if we could decide on a structure for the repository sooner. Then such things could be incorporated more smoothly.
